спринт 23

David-Arcanone opened the pull request:
примечание: текст АллертДиалогов в Фигма отличается от условий в тексте задания (не уверен по какому из критериев будут проверять, по фигме или заданию).
https://www.figma.com/design/CDSmqORU4qsKr3z5MALk41/(09.07.23)-Playlist-Maker-(YP)?node-id=1-345&t=Syl4xmEaoXv1axpV-0
-например в задании:
При нажатии на пункт «Удалить плейлист» всплывающее меню исчезает и пользователь видит диалог подтверждения с заголовком «Удалить плейлист», текстом «Хотите удалить плейлист?» и кнопками «Нет» и «Да»:
-а в фигме заголовка нет и текст отличается

David-Arcanone committed:
спринт 23
примечание: текст АллертДиалогов в Фигма отличается от условий в тексте задания (не уверен по какому из критериев будут проверять, по фигме или заданию).

David-Arcanone committed:
спринт 23
примечание: текст АллертДиалогов в Фигма отличается от условий в тексте задания (не уверен по какому из критериев будут проверять, по фигме или заданию).

kosstyango commented:
⚠️ Критические замечания:
Треки с ОЧЕНЬ длинным названием исполнителя показываются неправильно: без указания длительности трека. Проверь поиском слова "Метелица" - там таких треков будет два.
При выходе из экрана Аудиоплеер (если с этого трека пытались создавать новый плейлист) системной кнопкой Back появляется диалог "Завершить создание плейлиста?" - этого быть не должно.
Плейлист не удаляется после нажатия на кнопку "ДА" на диалоге подтверждения удаления плейлиста.
Треки в плейлисте хранятся НЕ в обратном порядке.
🍏 Рекомендации, не влияющие на результат проверки работы:
В сообщении "Трек уже добавлен в плейлист" нет пробела перед названием плейлиста.
На экране Плейлист кол-во минут в плейлисте указывается без пробела между цифрами и словом "минут"
Курсор в строке поиска должен быть СИНИЙ

David-Arcanone committed:
спринт 23
исправленные замечания:
-баг длинного названия исполнителя
-добавил обработчик onBackPressedDispatcher для фрагмента PlaylistOverview
-убрал лишний диалог при выходе из экрана редактирования плейлиста
-баг удаление плейлиста
-упорядочил треки для recycleview
-заменил курсор EditText на drawable синего цвета
-добавил отсутствующие пробелы (время_минуты, трек уже добавлен в плейлист_название)

David-Arcanone committed:
спринт 23
исправленные замечания:
-баг длинного названия исполнителя
-добавил обработчик onBackPressedDispatcher для фрагмента PlaylistOverview
-убрал лишний диалог при выходе из экрана редактирования плейлиста
-баг удаление плейлиста
-упорядочил треки для recycleview
-заменил курсор EditText на drawable синего цвета
-добавил отсутствующие пробелы (время_минуты, трек уже добавлен в плейлист_название)

+ реализовал кастомное оповещение вместо Toast

David-Arcanone committed:
спринт 23
исправленные замечания:
-баг длинного названия исполнителя
-добавил обработчик onBackPressedDispatcher для фрагмента PlaylistOverview
-убрал лишний диалог при выходе из экрана редактирования плейлиста
-баг удаление плейлиста
-упорядочил треки для recycleview
-заменил курсор EditText на drawable синего цвета
-добавил отсутствующие пробелы (время_минуты, трек уже добавлен в плейлист_название)

+ реализовал кастомное оповещение вместо Toast

kosstyango commented:
⚠️ Критические замечания:
При выходе из экрана Аудиоплеер (если с этого трека пытались создавать новый плейлист) системной кнопкой Back появляется диалог "Завершить создание плейлиста?" - **это пока осталось. (См.видео-скрин)**
https://github.com/user-attachments/assets/6b812952-690f-4e01-8c02-dee19eaa6eb6

🍏 Рекомендации, не влияющие на результат проверки работы:
Еще одна рекомендация - на экране Аудиоплеер треки с ОЧЕНЬ длинным названием не обрезаются и не переносятся: просто исчезают за краем экрана (этот пример тоже попал на видео-скрин про критическое замечание)

David-Arcanone committed:
спринт 23
исправленные замечания:
-баг длинного названия в аудиоплеере (сдвинул границу обрезания до правого Guideline, до этого был parent можно заметить ".." в вашем видео)
-баг с диалогом "Завершить создание плейлиста? (спасибо за видео, в первый раз я не понял, думал опечатка, и вы имели ввиду Диалог выхода при редактировании Плейлиста, который у меня был, но не должен был быть)

David-Arcanone committed:
спринт 23
исправленные замечания:
-баг длинного названия в аудиоплеере (сдвинул границу обрезания до правого Guideline, до этого был parent можно заметить ".." в вашем видео)
-баг с диалогом "Завершить создание плейлиста? (спасибо за видео, в первый раз я не понял, думал опечатка, и вы имели ввиду Диалог выхода при редактировании Плейлиста, который у меня был, но не должен был быть)
